learn/getting_started/quick_start.md

```diff
@@ -137,6 +137,10 @@ MeiliSearch **response**:
 }
 ```
 
+::: note
+By default, MeiliSearch returns the first **20 documents for a search query.** You can read more about the limit parameter [here](/reference/features/search_parameters.md#limit).
```

Comment:
```suggestion
**By default, MeiliSearch returns only the first 20 results** for a search query. You can read more about the limit parameter [here](/reference/features/search_parameters.md#limit).
```

Reply:
When picking which text to **bold**, I aim to make the bolded text readable by itself. It should be a complete clause that communicates a clear piece of information. That way someone who's scrolling quickly can still catch the pertinent information and stop to investigate if it's relevant to them.